api/bases/glance.openstack.org_glances.yaml

```diff
@@ -463,7 +463,6 @@ spec:
             required:
             - glanceAPIExternal
             - glanceAPIInternal
-            - secret
```

Comment:
lgtm, if this simplify your work on the meta-operator, it's fine by me.
FYI the next change which is coming for both cinder and glance (later on manila operator as well), is related to the introduction of a new API extension that allows to mount `extraVolumes`.
The CRD will include  something like the following:

```
 extraMounts:
    - propagation: All
      extraVolType: Ceph
      volumes:
      - name: ceph
        secret:
          secretName: ceph-client-conf
      mounts:
      - name: ceph
        mountPath: "/etc/ceph"
        readOnly: true
```

but this is unrelated to this change, so I'm go to go on this.